docs/getting-started/dev-quick-start/index.md

```diff
@@ -0,0 +1,342 @@
+       title: 'New heading',
+       showSubtitle: 'true',
+    -  overlineSize: 'hl-title__heading--size-5',
+       subtitle: 'New overline text',
+       'jcr:primaryType': 'nt:unstructured',
+       headingLevel: 'h1',
+       headingSize: 'hl-title__heading--size-2' }
+```
+
+Firstly, you updated the default font size of the overline to ensure consistency with the design. The first test detected the change and failed as a result. Secondly, you added a new property to the dialog for the component. It was recognised as well.
+
+
+### Update functional tests
+As functional tests fail due to the changes, you should adjust related assertions. They are placed in file `tests/end-to-end/tests/lunatitle.cy.ts`. 
+
+The first test checks the font size for the overline text. There are two component intances validated. Thus, you need to update assertions for both of them as follows.
+
+```typescript
+    cy.getByTestId('component_title1')
+      .findByTestId('overline')
+      .should('have.css', "font-size", "25.008px")
+      .should('have.text', 'Additional overline text filled')
+```
+
+```typescript
+    cy.getByTestId('component_title2')
+      .findByTestId('overline')
+      .should('have.css', "font-size", "25.008px")
+      .should('have.text', 'Resized to 6 cols on L breakpoint')
+```
+
+The second test validates the dialog for the component. Update the test to recognise the new input field.
+
+```typescript
+    cy.request(
+      '/content/luna-test/pages/LunaTitle/jcr:content/rootcontainer/maincontainer/pagesection/title.json'
+    )
+      .its('body')
+      .should('deep.eq', {
+        'sling:resourceType': 'luna/components/lunatitle',
+        title: 'New heading',
+        showSubtitle: 'true',
+        overlineSize: 'hl-title__heading--size-5',
+        subtitle: 'New overline text',
+        'jcr:primaryType': 'nt:unstructured',
+        headingLevel: 'h1',
+        headingSize: 'hl-title__heading--size-2'
+      });
+```
+
+### Running functional tests again
+
+Now, you can execute the functional tests again.
+
+```shell
+mvn -f tests/content/pom.xml clean install -P autoInstallPackage
+npm run-script test --prefix tests/end-to-end
+```
+
+Both tests should passs this time. You should get a report like the one below.
+```
+Running:  lunatitle.cy.ts                                                                 (1 of 1)
+ 
+ 
+Luna Title component
+  ✓ renders correctly in preview mode (1030ms)
+  ✓ renders correctly in edit mode (2774ms)
+
+
+2 passing (4s)
+
+```
+
+Congratulations! You updated the component, and it passed tests.
+
+## Part E: Use the new component
+
+!!! info "Hint"
+    If you need help to navigate inside WebSight see the general [getting started](/docs/getting-started/quick-start/index.md) for details.
+
+The page owner can use the updated component now. Let's check it too.
+
+Run WebSight, open _Luna_ space and edit home page. Find the _Title_ having text `Meet our New Grand Luxor Jewelry Collection`. 
```

Comment:
I'd skip writing about `Run WebSight` - it should be already running according to the tutorial.
Maybe
> Navigate to the spaces panel and open _Luna_ space...

would be better?

Reply:
Previous parts focus on code changes. The goal here is to provide a context on where to execute the next steps. I would keep it.

Reply:
Open replaced by Switch